basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronOperations.java

```diff
@@ -0,0 +1,55 @@
+package org.jboss.hal.testsuite.test.configuration.elytron;
+
+import static org.wildfly.extras.creaper.core.online.Constants.EXTENSION;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+public class ElytronOperations {
+
+    private static final String ELYTRON_EXTENSION_NAME = "org.wildfly.extension.elytron";
+    private static final Address
+        ELYTRON_EXTENSION_ADDRESS = Address.of(EXTENSION, ELYTRON_EXTENSION_NAME),
+        ELYTRON_SUBSYSTEM_ADDRESS = Address.subsystem("elytron");
+    static final String
+        PROVIDER_LOADER = "provider-loader";
+
+    private final Operations ops;
+    private final Administration adminOps;
+
+    public ElytronOperations(OnlineManagementClient client) {
+        this.ops = new Operations(client);
+        this.adminOps = new Administration(client);
+    }
+
+    public void enableElytronSubsystem() throws IOException, InterruptedException, TimeoutException {
+        ops.add(ELYTRON_EXTENSION_ADDRESS, Values.of("module", ELYTRON_EXTENSION_NAME));
```

Comment:
Can you please add `ModelNodeResult#assertSuccess()` to be sure that operation adding Elytron subsystem went well?